.github/scripts/requirements.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+# Constants
+source $GITHUB_WORKSPACE/.github/scripts/constants.sh
+
+function install_igluctl() {
+    if [ ! -f "${IGLUCTL}" ]
+    then
+        echo "igluctl not found, downloading..."
+        wget "${VALIDATOR_URI}" -P "${INSTALL_DIR}"
```

Comment:
Just to confirm this is downloading the JAR directly and we are certain it will be executable?  Would it be worth adding a `chmod +x igluctl` here just in case a binary gets uploaded that does not have the executable bit set?

Reply:
Hey @jbeemster! No, it won't be executable, we run it everywhere with `java -jar ${IGLUCTL}`, so we're certain it will execute it.

Reply:
Easy done.